jdaviz/configs/default/plugins/subset_tools/subset_tools.vue

```diff
@@ -53,10 +69,25 @@
              :selected.sync="dataset_selected"
```

Comment:
should be 
`:items="recenter_dataset_items"`
`:selected.sync="recenter_dataset_selected"`

Reply:
good catch 😬 should be fixed now

Reply:
Thank you, @cshanahan1, for making the post. That was the only thing I had seen as well; everything else looks good to me. Thank you, @kecnry, for your work!